src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure.Tests/CloudErrorJsonConverterTest.cs

```diff
@@ -93,5 +104,82 @@ public void TestEmptyCloudErrorDeserialization()
             Assert.NotNull(cloudError);
             Assert.Null(cloudError.Code);
         }
+
+        [Fact]
```

Comment:
@j5lim this test is good, but after the change requested of removing actual violation type, you might have to rewrite the asserts